heron/schedulers/src/java/com/twitter/heron/scheduler/slurm/SlurmController.java

```diff
@@ -140,7 +137,7 @@ public boolean killJob(String jobIdFile) {
     List<String> jobIdFileContent = readFromFile(jobIdFile);
     if (jobIdFileContent.size() > 0) {
       String[] slurmCmd = new String[]{"scancel", jobIdFileContent.get(0)};
-      return runProcess(null, slurmCmd, new StringBuilder(), new StringBuilder());
+      return runProcess(null, slurmCmd, new StringBuilder());
```

Comment:
Why does this method call require passing a StringBuilder that the caller never uses? Can we simplify the API to not require this here, maybe with another signature?